### PR TITLE
fix fivem startup issues

### PR DESCRIPTION
**Pull request by Arisstath**
### All Submissions:

* [ ] Have you followed the guidelines in our Contributing document? Yes
* [ ] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change? Yes
* [ ] Did you branch your changes and PR from that branch and not from your master branch?
  * If not, why?:

### Changes to an existing Egg:

1. [ ] Have you added an explanation of what your changes do and why you'd like us to include them? Yes
2. [ ] Have you tested your Egg changes? Yes

image

Special thanks to #project-discussion and **SeaLife#7722**.

### CODE REVIEW

**Comment by SeaLife** [comment]
This PR contains more background information, so i closed mine.

Fixes #1268 #1187 #1235 #1224 #1220 #1215 #1209 #1188 probably.

File: game_eggs/gta/fivem/egg-five-m.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
         {
             "name": "fivem license",
             "description": "Required to start the service. Get your keys at https:\/\/keymaster.fivem.net\/",
-            "env_variable": "FIVEM_LICENSE",
+            "env_variable": "sv_licenseKey",
```
Comment (SeaLife) [inline game_eggs/gta/fivem/egg-five-m.json:34]: This requires to be `FIVEM_LICENSE`
If you wanna rename this variable here, you need to change the "files"-part where the license is being replaced in the server config AND remove the unset-command

**Comment by blattersturm** [comment]
You are aware acts like this (and the continued trolling from your community on our Discord chat) _will not resolve the problem you folks are having with us_, right?

Instead you'll just harm yourself _and everyone else_ in this process.

**Comment by SeaLife** [comment]
> You are aware acts like this (and the continued trolling from your community on our Discord chat) _will not resolve the problem you folks are having with us_, right?
> 
> Instead you'll just harm yourself _and everyone else_ in this process.
> 
> If you're as mature a community as you claim you are, **stop this targeted harassment immediately**.

I guess, we'r just fixing an issue here for the whole Pterodactyl-Community which should not be an issue at all. The FiveM Community is toxic (especially the core members) to like everyone, not just Pterodactyl.

Its not pterodactyl's fault to let people host a free-to-self-host fivem server. If companies uses Pterodactyl to break you'r TOS, your having a proble with those companies not with Pterodactyl. And thats the main issue here.

There are many people using Pterodactyl privately to host FiveM. And your being childish creating code to block that?

**Comment by Software-Noob** [comment]
The egg wil not be updated to bypass any checks as Parker has agreed.

**Comment by blattersturm** [comment]
> The egg wil not be updated to bypass any checks as Parker has agreed.

Thanks. Again, we are still intending to add explicit functionality to allow people to use hosting panels _on explicit approval and review that it's being used for personal use_, however the time frame of this is still unknown.

**Comment by blattersturm** [comment]
> There are many people using Pterodactyl privately to host FiveM.

Also, this PR's author seems to be _owning a game server hosting company_ which also coincidentally offers services for 'Rage MP' which is a (less transparent and somewhat less used) competitor of FiveM, in a market where there should be collaboration rather than competition and hostility.

**Comment by Arisstath** [comment]
> You are aware acts like this (and the continued trolling from your community on our Discord chat) _will not resolve the problem you folks are having with us_, right?
> 
> Instead you'll just harm yourself _and everyone else_ in this process.
> 
> If you're as mature a community as you claim you are, **stop this targeted harassment immediately**. It also doesn't help when core members of your project act **the exact same as the troll army you folks send**. The ban of this 'Tactical Fish' person from our chat the other day was _solely_ based on those signals, the same as you _are still continuing to send_.



> > The egg wil not be updated to bypass any checks as Parker has agreed.
> 
> Thanks. Again, we are still intending to add explicit functionality to allow people to use hosting panels _on explicit approval and review that it's being used for personal use_, however the time frame of this is still unknown.

I Hope take-two is okay with you guys policing their assets. Wish you all the luck.

**Comment by SeaLife** [comment]
> > The egg wil not be updated to bypass any checks as Parker has agreed.
> 
> Thanks. Again, we are still intending to add explicit functionality to allow people to use hosting panels _on explicit approval and review that it's being used for personal use_, however the time frame of this is still unknown.

You'r talking non-sense their. Its not abour YOUR approval to allow ME to use any managed software to host my game servers. No one can forbid me to use any panel i like. I respect the decisions made by the core team here, but i'll continue using Pterodactyl with any workaround i can. I pay for my license and if you now decide to restrict us even more, its probably on the downside for you guys.
I wont discuss this much further on this issue, because its not an issue with Pterodactyl, but an Problem with your "policy" which is not even legal in most EU countries anyway.

**Comment by blattersturm** [comment]
> but an Problem with your "policy" which is not even legal in most EU countries anyway.

Just as I keep seeing this narrative appear every so often, are you able to cite _any_ law that prohibits exclusive distribution agreements?

**Comment by Software-Noob** [comment]
If you wish to continue any discussion, do it through another method, such as Discord DMs. Please do not attack anyone or stir up drama here.

**Pull request closed without merging**